articles/connections/social/devkeys.md

```diff
@@ -28,6 +28,6 @@ When using the Auth0 developer keys, there are a few caveats you need to be awar
 
 4. [Federated Logout](/logout#log-out-a-user) does not work. When using the Auth0 developer keys, calling `/v2/logout?federated` will sign the user out of Auth0, but not out of the Social Identity Provider.
 
-5. `prompt=none` won't work on the [/authorize](/api/authentication/reference#social) endpoint. [Auth0.js' `silentAuthentication` method](https://github.com/auth0/auth0.js#silent-authentication) uses `prompt=none` internally, so that won't work either.
```

Comment:
Yeah, I'd either change the method name to `renewAuth` and link to the GitHub - or to the docs page here https://auth0.com/docs/libraries/auth0js#using-renewauth-to-acquire-new-tokens (I'll make a card to strengthen the renewAuth section now that there's more info on it), or you could link to both the v7 and v8 methods. I'd probably stick with the v8 method, myself, but both are supported so that's an option. Definitely don't just leave it with the v7 one though.